proto/dydxprotocol/listing/params.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package dydxprotocol.listing;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/listing/types";
+
+// ListingVaultDepositParams represents the params for PML megavault deposits
+message ListingVaultDepositParams {
+  // Amount that will be deposited into the new market vault
+  bytes new_vault_deposit_amount = 1 [
+    (gogoproto.customtype) =
+        "github.com/dydxprotocol/v4-chain/protocol/dtypes.SerializableInt",
+    (gogoproto.nullable) = false
+  ];
+
+  // Amount deposited into the main vault
```

Comment:
could you specify in the comment whether  `new_vault_deposit_amount` is part of `main_vault_deposit_amount`